dashboard/test/models/lesson_test.rb

```diff
@@ -806,7 +806,7 @@ class LessonTest < ActiveSupport::TestCase
 
     assert_equal(
       new_lesson.start_url,
-      "//test-studio.code.org/s/#{new_script.name}/lockable/1/levels/1"
+      CDO.studio_url("/s/#{new_script.name}/lockable/1/levels/1", CDO.default_scheme)
```

Comment:
this will work, but I think the best practice would be to use `https://test-studio.code.org/` here, so that the expected value is not dependent on the implementation of studio_url or default_scheme -- otherwise our test code unnecessarily depends on part of our implementation. unit tests always run in the test environment whether they are running in drone, DTT or on your local machine, so this should be safe in terms of tests passing. 
```suggestion
      "https://test-studio.code.org/s/#{new_script.name}/lockable/1/levels/1"
```

Reply:
Oh, for some reason I was thinking that it would be different when run in drone than when run locally which is why I wrote the test like this. Will update!

Also - no worries on the back-and-forth, I'm glad we came to the best solution in the end